R/wflow_start_rstudio.R

```diff
@@ -0,0 +1,49 @@
+## Wrapper function to enhance RStudio Project Template
+
+wflow_start_rstudio <- function(directory,
+                                name = NULL,
```

Comment:
Since we know that RStudio will pass `name`, `user.name`, and `user.email` as character vectors, I think it would be more transparent to make the default argument the same, e.g. `name = ""`. Thoughts?

Reply:
I agree. I initially have the arguments same as wflow_start (i.e. copy and paste from there), but it make more sense to set the default as empty string.

I have made the edits in the latest commit.